packages/editor/src/components/parts/name/NamePart.tsx

```diff
@@ -12,20 +12,26 @@ export function useNamePart(options?: { hideTags?: boolean }): PartProps {
     name: 'Name',
     state,
     reset: { dirty, action: () => resetData() },
-    content: <NamePart hideTags={options?.hideTags} />,
+    content: <NamePart hideTags={options?.hideTags} disableName={options?.disableName} />,
     summary: <NameSummary hideTags={options?.hideTags} />
   };
 }
 
-const NamePart = (props: { hideTags?: boolean }) => {
+const NamePart = (props: { hideTags?: boolean; disableName?: boolean }) => {
   const { data, update } = useNameData();
 
   const nameField = useFieldset();
   const descriptionField = useFieldset();
   return (
     <>
       <Fieldset label='Display name' {...nameField.labelProps}>
-        <Textarea maxRows={3} value={data.name} onChange={change => update('name', change)} {...nameField.inputProps} />
+        <Textarea
+          maxRows={3}
+          disabled={props.disableName ? true : false}
```

Comment:
you can also do `!!props.disableName` 
this is a duplicated negation, so if it is true or false, it will be again the same
if it is undefined -> first negation: true -> second negation: false